website/docs/reference/supported_commands.md

```diff
@@ -401,10 +428,89 @@ db.aggregate()
+|                                    |                                | `expireAfterSeconds`      | ⚠️      |                                                           |
+|                                    |                                | `hidden`                  | ⚠️      |                                                           |
+|                                    |                                | `storageEngine`           | ⚠️      |                                                           |
+|                                    |                                | `weights`                 | ⚠️      |                                                           |
+|                                    |                                | `default_language         | ⚠️      |                                                           |
+|                                    |                                | `language_override`       | ⚠️      |                                                           |
+|                                    |                                | `language_override`       | ⚠️      |                                                           |
+|                                    |                                | `2dsphereIndexVersion`    | ⚠️      |                                                           |
+|                                    |                                | `bits`                    | ⚠️      |                                                           |
+|                                    |                                | `min`                     | ⚠️      |                                                           |
+|                                    |                                | `max`                     | ⚠️      |                                                           |
+|                                    |                                | `bucketSize`              | ⚠️      |                                                           |
+|                                    |                                | `collation`               | ⚠️      |                                                           |
+|                                    |                                | `wildcardProjection`      | ⚠️      |                                                           |
+|                                    |                                | `min`                     | ⚠️      |                                                           |
+|                                    |                                | `max`                     | ⚠️      |                                                           |
+|                                    |                                | `bucketSize`              | ⚠️      |                                                           |
```

Comment:
These are already listed above, no need to duplicate them :hugs: 
```suggestion
```

Reply:
Listed above for `create` or the update operators? These are arguments specifically for 2d indexes, they define bounds and behave different from the `$min` / `$max` operators. Should we keep these separate? Maybe I am confused.

Reply:
See 505-507 is repeated in 510-512. They are all under `createIndexes` `indexes` so I think we can remove the duplicated `min`,`max` and `bucketSize` right? :hugs:

Reply:
Ah I see! I fixed that in newest commit. Thanks!